test/postman_test/SSAS.postman_collection.json

```diff
@@ -1,10 +1,378 @@
 {
 	"info": {
-		"_postman_id": "769bc51d-c8cc-4e9e-828b-4e9372b24414",
+		"_postman_id": "4b27bbbe-fe8c-4195-a852-7644248dd722",
 		"name": "SSAS",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
+		{
+			"name": "admin post group, no group_id",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "1678ec38-847b-4912-9d74-469a233d0200",
+						"exec": [
+							"pm.test(\"response is 400\", function () {",
+							"    pm.response.to.have.status(400);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"name": "Content-Type",
+						"value": "application/json",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"bad-json\": \"is-bad\"\n}"
+				},
+				"url": {
+					"raw": "{{scheme}}://{{host}}:{{admin-port}}/group",
+					"protocol": "{{scheme}}",
+					"host": [
+						"{{host}}"
+					],
+					"port": "{{admin-port}}",
+					"path": [
+						"group"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "admin post group, success",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "20ed753f-bd64-4270-9993-305d4efc1372",
+						"exec": [
+							"pm.test(\"response is created and returns json\", function () {",
+							"    pm.response.to.have.status(201);",
+							"    pm.response.to.be.json;",
```

Comment:
Checking the response body could be helpful.

Reply:
i am now checking the response body!  thanks